src/client/pps.go

```diff
@@ -588,17 +588,13 @@ func (c APIClient) StopPipeline(name string) error {
 	return grpcutil.ScrubGRPC(err)
 }
 
-// RerunPipeline reruns a pipeline over a given set of commits. Exclude and
-// include are filters that either include or exclude the ancestors of the
-// given commits.  A commit is considered the ancestor of itself. The behavior
-// is the same as that of ListCommit.
-func (c APIClient) RerunPipeline(name string, include []*pfs.Commit, exclude []*pfs.Commit) error {
-	_, err := c.PpsAPIClient.RerunPipeline(
+// RunPipeline runs a pipeline over a given set of commits.
```

Comment:
I think this comment should be expanded a bit to mention that `provenance` is the input commits, most people using Pachyderm will know that by the time they get to this function, but I think it's good to reiterate that concept a lot for people. It'd also probably be good to mention that you don't need to pass provenance for all of your inputs, or any of them for that matter. `RunPipeline(pipeline, nil)` will just trigger a job with the head commits right?